modules/configuring-the-collector-to-receive-audit-logs-as-an-http-server.adoc

```diff
@@ -0,0 +1,129 @@
+// * configuring/cluster-logging-collector.adoc
+
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-08-05
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-the-collector-to-receive-audit-logs-as-an-http-server_{context}"]
+= Configuring the collector to receive audit logs as an HTTP server
+
+You can configure your log collector to listen for HTTP connections to only receive audit logs by specifying `http` as a receiver input in the `ClusterLogForwarder` custom resource (CR).
+
+:feature-name: HTTP receiver input
+include::snippets/logging-http-sys-input-support.adoc[]
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Modify the `ClusterLogForwarder` CR to add configuration for the `http` receiver input:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  namespace: <control_plane_namespace>
+# ...
+spec:
+  serviceAccount:
+    name: <service_account_name>
+  inputs:
+  - name: http-receiver #<1>
+    type: receiver
+    receiver:
+      type: http #<2>
+      port: 8443 #<3>
+      http:
+        format: kubeAPIAudit #<4>
+  outputs:
+  - name: http-output
+    type: http
+    http:
+      url: <url>
+  pipelines: #<5>
+    - name: http-pipeline
+      inputRefs:
+        - http-receiver
+      outputRefs:
+        - my-http-output
+# ...
+----
+<1> Specify a name for your input receiver.
+<2> Specify the input receiver type as `http`.
+<3> Optional: Specify the port that the input receiver listens on. This must be a value between `1024` and `65535`. The default value is `8443` if this is not specified.
+<4> Currently, only the `kube-apiserver` webhook format is supported for `http` input receivers.
+<5> Configure a pipeline for your input receiver.
+
+. Apply the changes to the `ClusterLogForwarder` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Verify that the collector is listening on the service that has a name in the `<clusterlogforwarder_resource_name>-<input_name>` format by running the following command:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
+----
+NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)            AGE
+collector                 ClusterIP   172.30.85.239    <none>        24231/TCP          3m6s
+collector-http-receiver   ClusterIP   172.30.205.160   <none>        8443/TCP           3m6s
+----
++
+In the example, the service name is `collector-http-receiver`.
+
+.Verification
+
+. Extract the certificate authority (CA) certificate file by running the following command:
++
+[source,terminal]
+----
+$ oc extract cm/openshift-service-ca.crt -n <namespace>
+----
+
+. Create `kubeconfig` configuration used by the {hcp} by running the following command:
```

Comment:
Is this really needed? it seems to hard to be needed this step as this can break the cluster is an error is done generating the kubeconfig and it could implies also a restart of the nodes.

Is not the application that log forward the logs to the receiver a pod or does it not allow to configure the certificates reading from a configmap/secret or file?